packages/compass-connections/src/components/connection-status-notifications.tsx

```diff
@@ -192,10 +186,8 @@ const openConnectionFailedToast = ({
   // can happen is autoconnect flow
   connectionInfo: ConnectionInfo | null | undefined;
   error: Error;
-  showReviewButton: boolean;
-  showDebugButton: boolean;
-  onReviewClick: () => void;
-  onDebugClick: () => void;
+  onReviewClick?: () => void;
```

Comment:
personal preference but I think since showing and the handler are inherently tied it's good to combine the two. It's a pattern in LG too so works out pretty well usually